src/components/SummaryBox/SummaryBox/SummaryBox.test.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react'
+import { getByRole, render } from '@testing-library/react'
```

Comment:
This one also

Reply:
Done. I'm surprised eslint doesn't find this problematic enough to fail the CI check.

Reply:
Good point. I think that unused direction might be a warning not error? Perhaps we need to adjust to have CI checks fail on warnings too - I think its just adding `--max-warnings=0` to our eslint call.